specification/security/disable_user/SecurityDisableUserRequest.ts

```diff
@@ -21,6 +21,8 @@ import { RequestBase } from '@_types/Base'
 import { Refresh, Username } from '@_types/common'
 
 /**
+ * Deactivate users.
```

Comment:
I'd go with enable/disable here (as well as for the profile routes) since that matches the existing API, ES docs, and its terminology that's been around for a while now. Not a string opinion though.

Reply:
Yes, it make sense to stay consistent for now. I've made that change in dc8609aa4cc030bb67a447c204418f23807ce143